src/value.h

```diff
@@ -146,18 +142,24 @@ void RSValue_Clear(RSValue *v);
  * the actual value object */
 void RSValue_Free(RSValue *v);
 
-
-RSValue* RSValue_IncrRef(RSValue* v);
-void RSValue_Decref(RSValue* v);
+RSValue *RSValue_IncrRef(RSValue *v);
+void RSValue_DecrRef(RSValue *v);
 
 RSValue *RS_NewValue(RSValueType t);
 
+/**
+ * Creates a static undefined RSValue.
+ * The returned value is not allocated on the heap and should not be freed.
+ * @return An undefined RSValue with static storage
+ */
+RSValue RSValue_Undefined_Static();
```

Comment:
I would say is better Stack than Static? It does not necessarily lives during the complete duration of the program right?